build_node/builders/base_rpm_builder.py

```diff
@@ -262,8 +263,13 @@ def build_packages(self, src_dir, spec_file=None):
             if srpm_build_result:
                 self.save_build_artifacts(srpm_build_result,
                                           srpm_artifacts=True)
-        srpm_path = srpm_build_result.srpm
-        self.logger.info('src-RPM %s was successfully built', srpm_path)
+        if srpm_build_result:
+            srpm_path = srpm_build_result.srpm
+            self.logger.info('src-RPM %s was successfully built', srpm_path)
+        elif self.task.built_srpm_url:
+            srpm_path = built_srpm_url
```

Comment:
`srpm_path` is possibly unbound and `built_srpm_url` is undefined

Reply:
Right, I meant self.task.build_srpm here.